modules/nomad-clients/main.tf

```diff
@@ -1,11 +1,5 @@
 locals {
-  allowed_inbound_cidr_blocks = concat(list(data.aws_vpc.selected.cidr_block), var.allowed_inbound_cidr_blocks)
-  services_inbound_cidr       = concat(list(data.aws_vpc.selected.cidr_block), var.nomad_clients_services_inbound_cidr)
```

Comment:
Can i check if the reason `nomad_clients_services_inbound_cidr` was including the VPC's CIDR block is because it assumes that only one VPC per nomad client cluster?

Reply:
Not so much one VPC per nomad client cluster, but rather, the original set-up assumes there is only one IP CIDR (the CIDR spans the entire allows range for the VPC basically). While this is typically true for VPC that are non-restrained, it is always the case for every general VPC.

For info, a VPC can contain multiple non-contiguous IP CIDRs, and one might not want every CIDR blocks in the VPC to be allowed access to Nomad. Thus removing this assumption allows the derivative module to put whatever CIDRs it prefers.